stdlib/public/core/StringGuts.swift

```diff
@@ -913,20 +980,23 @@ extension _StringGuts {
 
     // Copy ourselves in
     self._copy(
+      range: 0..<self.count,
       into: buffer.start,
       capacityEnd: buffer.capacityEnd,
       accomodatingElementWidth: byteWidth)
     return buffer
   }
 
   @_inlineable
-  // TODO: @_versioned
-  // TODO: internal
+  @_transparent
   public // TODO(StringGuts): for testing only
   mutating func isUniqueNative() -> Bool {
     return _isUnique(&_storage.0) && _isNative
```

Comment:
Do the lessons from https://github.com/apple/swift/pull/13000 apply here as well?